apps/api/src/server/applications/application/representations/representation.mapper.js

```diff
@@ -85,3 +85,20 @@ export const mapCreateOrUpdateRepRequestToRepository = (
  */
 export const getLatestRedaction = ({ representationActions }) =>
```

Comment:
I know this is existing code but can we guarantee this is a redaction and not a status update?

Reply:
done - 

export const getLatestRedaction = ({ representationActions }) =>
	representationActions
		.filter(({ type }) => type === 'REDACTION')
		.slice(-1)
		.pop();